src/software/simulated_tests/non_terminating_validation_functions/robots_in_friendly_half_validation.h

```diff
@@ -4,9 +4,18 @@
 #include "software/world/world.h"
 
 /**
- * Checks if the friendly robots are in friendly half of the field, else assertion fails.
+ * Checks if the friendly robots are in the friendly half of the field, else assertion
+ * fails.
  * @param world_ptr the world pointer given by the simulator. Gets updated every tick
  * @param yield yields control to the next routine (coroutines)
  */
 void robotsInFriendlyHalf(std::shared_ptr<World> world_ptr,
                           ValidationCoroutine::push_type& yield);
+
+/**
+ * Checks if a friendly robot is in the friendly half of the field, else assertion fails.
+ * @param world_ptr the world pointer given by the simulator. Gets updated every tick
+ * @param yield yields control to the next routine (coroutines)
+ */
```

Comment:
Nit: Is there a reason `robot` isn't included in the `@param` list? I noticed `robotID` is included in the `@param` list below so I'm wondering if there's an inconsistency.

Reply:
Added